hazelcast/src/main/java/com/hazelcast/map/impl/proxy/MapProxyImpl.java

```diff
@@ -787,14 +787,12 @@ public ICompletableFuture submitToKey(K key, EntryProcessor entryProcessor) {
     public <R> R aggregate(Aggregator<Map.Entry<K, V>, R> aggregator, Predicate<K, V> predicate) {
         checkNotNull(aggregator, NULL_AGGREGATOR_IS_NOT_ALLOWED);
         checkNotNull(predicate, NULL_PREDICATE_IS_NOT_ALLOWED);
+        checkNotPagingPredicate(predicate, "aggregate");
```

Comment:
This is just a unification in line 790. There was already an exception thrown in line 795